homeassistant/components/bsblan/climate.py

```diff
@@ -138,7 +138,7 @@ def current_temperature(self):
     @property
     def hvac_mode(self):
         """Return the current operation mode."""
-        return self._current_hvac_mode
```

Comment:
`self._current_hvac_mode` is no longer used and can be removed from the init method.

Reply:
thanks